pkg/services/tag/tagimpl/tag.go

```diff
@@ -4,23 +4,14 @@ import (
 	"context"
 
 	"github.com/grafana/grafana/pkg/infra/db"
-	"github.com/grafana/grafana/pkg/services/featuremgmt"
 	"github.com/grafana/grafana/pkg/services/tag"
-	"github.com/grafana/grafana/pkg/setting"
 )
 
 type Service struct {
 	store store
 }
 
-func ProvideService(db db.DB, cfg *setting.Cfg) *Service {
-	if cfg.IsFeatureToggleEnabled(featuremgmt.FlagNewDBLibrary) {
```

Comment:
@zserge @papagian -- removing the sqlx tag implementation got added here since the cascade of changing most test files came from this 😬 

can split out to its own PR, but seems trivial enough